controllers/managedfusionoffering_controller.go

```diff
@@ -180,6 +181,9 @@ func (r *ManagedFusionOfferingReconciler) reconcilePhases() (reconcile.Result, e
 		if err := r.reconcileSubscription(); err != nil {
 			return ctrl.Result{}, err
 		}
+		if err := reconcileOfferingConfig(r.managedFusionOffering); err != nil {
+			return ctrl.Result{}, err
+		}
```

Comment:
The function shouldn't be named `reconcile*` as we are not reconciling anything in the function. Why are we calling it after creating operatorGroup, CatSrc and Subscription?
The config should be validated just after we get ManagedFusionOffering CR. According to me, if the config is invalid then there is no point in creating operatorGroup, catSrc and Subscription.

Reply:
renamed func to validateOfferingConfig and set the calling before operatorGroup